tools/git_lfs.rst

```diff
@@ -34,6 +34,19 @@ Git LFS requires Git version 1.8.2 or later to be installed.
 
 Before you can use Git LFS with LSST data you'll need to configure by following the next section.
 
+.. _git-lfs-basic-config:
+
+Basic Git LFS configuration
```

Comment:
Should this section be listed under "On this page" above? The other section headings at this level are.

Actually, it might make more sense to have a single "configuring Git LFS" section with "basic" and "for LSST" as subsections.